cmd/metricchecker/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"encoding/base64"
+	"io/ioutil"
+	"log"
+	"time"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/PingCAP-QE/metrics-checker/pkg/metric"
+)
+
+var (
+	address        string
+	configFilePath string
+	configBase64   string
+	grafanaAPIURL  string
+	config         Config
+)
+
+type Config struct {
+	startTime     time.Time
+	StartAfter    time.Duration
+	Interval      time.Duration
+	Rules         []metric.Rule
+	MetricsToShow map[string]string
+}
+
+func (r *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var tmp struct {
+		StartAfter    string            `yaml:"start-after,omitempty"`
+		Interval      string            `yaml:"interval,omitempty"`
+		Rules         []metric.Rule     `yaml:"rules"`
+		MetricsToShow map[string]string `yaml:"metrics-to-show"`
+	}
```

Comment:
Why not use `Config`?

Reply:
> whether we should regard Config as a direct reflection of configuration file.

I think the answer is yes.
https://github.com/pingcap/tidb/blob/9e3c0649e43db48816e005e76c1c9d6bdd5a80ad/config/config.go#L86